### PR TITLE
Fix/lint

### PR DESCRIPTION
**Pull request by chad11111**
## Check List

- [ ] The title of my pull request is a short description of the changes
- [ ] This PR relates to some issue: 
- [ ] I have documented the changes made (if applicable)
- [ ] I have covered the changes with unit tests

### CODE REVIEW

File: dashboard/src/components/Badge.tsx [hunk 1]
```diff
@@ -37,12 +37,12 @@ export interface BadgeProps {
...
-    [BadgeCodes.ERROR]: "bg-failed text-white",
-    [BadgeCodes.WARNING]: "bg-[#ffa800] text-white",
-    [BadgeCodes.SUCCESS]: "bg-[#00c2ab] text-black-50 text-black-800",
+    [BadgeCodes.ERROR]: "bg-red-500 text-white",
+    [BadgeCodes.WARNING]: "bg-yellow-400 text-white",
+    [BadgeCodes.SUCCESS]: "bg-green-300 text-black-50 text-black-800",
     [BadgeCodes.INFO]: "bg-blue-200 text-black-800",
     [BadgeCodes.DEFAULT]: "text-black-800",
-    [BadgeCodes.UNKNOWN]: "text-failed bg-zinc-200",
+    [BadgeCodes.UNKNOWN]: "text-red-500 bg-zinc-200",
```
Comment (nir2002) [inline dashboard/src/components/Badge.tsx:45]: no
1. these are not the right colors, why did you changed them?
2. use tailwind theme colors

File: dashboard/src/API/releases.ts [hunk 2]
```diff
@@ -286,10 +286,10 @@ export function useChartReleaseValues({
 
 // Request objects
 interface ReleaseInfoParams {
-  chart: string;
+  chart?: string;
   tab: string;
-  namespace: string;
-  revision: string;
+  namespace?: string;
+  revision?: string;
```
Comment (nir2002) [inline dashboard/src/API/releases.ts:292]: does this make sense?

File: dashboard/src/components/ClustersList.tsx [hunk 3]
```diff
@@ -133,7 +133,6 @@ function ClustersList({
               onChange={(event) => {
                 onNamespaceChange(event.target.value);
               }}
-              checked={filteredNamespaces?.includes(namespace.name)}
```
Comment (nir2002) [inline dashboard/src/components/ClustersList.tsx:136]: did you made sure all of the functionality regarding namespaces works as expected?
install chart, delete chart, upgrade, downgrade, etc.

File: dashboard/src/components/revision/RevisionDiff.tsx [hunk 4]
```diff
@@ -111,16 +111,16 @@ function RevisionDiff({ includeUserDefineOnly }: RevisionDiffProps) {
...
         configuration
       );
       diff2htmlUi.draw();
       diff2htmlUi.highlightCode();
     } else if (viewMode === VIEW_MODE_VIEW_ONLY) {
-      diffElement.current.innerHTML = "";
+      diffElement!.current!.innerHTML = "";
     } else if (fetchedDataSuccessfully && (!hasRevisionToDiff || !data)) {
-      diffElement.current.innerHTML = "No differences to display";
+      diffElement!.current!.innerHTML = "No differences to display";
```
Comment (nir2002) [inline dashboard/src/components/revision/RevisionDiff.tsx:123]: lets not do this, its bad practice

File: dashboard/src/components/revision/RevisionDiff.tsx [hunk 5]
```diff
@@ -52,7 +52,7 @@ function RevisionDiff({ includeUserDefineOnly }: RevisionDiffProps) {
 
   const additionalParams = useMemo(() => {
     let additionalParamStr = "";
-    if (!!userDefinedValue) {
+    if (userDefinedValue) {
```
Comment (nir2002) [inline dashboard/src/components/revision/RevisionDiff.tsx:55]: wrap it with Boolean

File: dashboard/src/components/revision/RevisionResource.tsx [hunk 6]
```diff
@@ -90,9 +90,9 @@ const ResourceRow = ({
         <td className={cellClassnames}>
           {reason ? <Badge type={badgeType}>{reason}</Badge> : null}
         </td>
-        <td className={"rounded text-gray-400 " + cellClassnames}>
+        <td className={"rounded text-gray-100" + cellClassnames}>
           <div className="flex flex-col space-y-1 justify-start items-start">
-            {message && <div>{message}</div>}
+            {message && <div style={{ color: "grey", fontWeight: 100}}>{message}</div>}
```
Comment (nir2002) [inline dashboard/src/components/revision/RevisionResource.tsx:95]: why are we using inline style? use tailwind

File: dashboard/src/layout/Header.tsx [hunk 7]
```diff
@@ -48,7 +48,7 @@ export default function Header() {
   const getBtnStyle = (identifier: string) =>
     `text-md py-2.5 px-5 ${
       location.pathname.includes(`/${identifier}`)
-        ? " text-[#1347FF]  bg-[#EBEFFF]"
+        ? " text-[#1347FF] rounded-sm bg-[#EBEFFF]"
```
Comment (nir2002) [inline dashboard/src/layout/Header.tsx:51]: use our theme colors please

File: dashboard/src/pages/Installed.tsx [hunk 8]
```diff
@@ -51,7 +51,8 @@ function Installed() {
     return (
       data?.filter((installedPackage: Release) => {
         return (
-          installedPackage.name.includes(filterKey) &&
+          installedPackage.name.includes(filterKey) && 
+          installedPackage.namespace.includes(filterKey) &&
```
Comment (nir2002) [inline dashboard/src/pages/Installed.tsx:55]: you sure? please explain

File: dashboard/src/pages/Repository.tsx [hunk 9]
```diff
@@ -55,7 +55,7 @@ function RepositoryPage() {
         onRepositoryChanged={handleRepositoryChanged}
         selectedRepository={selectedRepository}
       />
-      <div className="w-10/12">
+      <div className="w-10/12" style={{width: "1014px"}}>
```
Comment (nir2002) [inline dashboard/src/pages/Repository.tsx:58]: what the hell?
you have tailwind! and we already use it! w-10/12
don't use inline styles!

**Review by nir2002 — CHANGES_REQUESTED** [review]

File: dashboard/src/pages/Installed.tsx (hunk 8 above)
Comment (chad11111) [reply dashboard/src/pages/Installed.tsx:55]: filter namespace as well

Comment (chad11111) [reply dashboard/src/pages/Installed.tsx:55]: needs to be || tho ill fix

File: dashboard/src/layout/Header.tsx (hunk 7 above)
Comment (chad11111) [reply dashboard/src/layout/Header.tsx:51]: im making a theme pr

File: dashboard/src/components/ClustersList.tsx (hunk 3 above)
Comment (chad11111) [reply dashboard/src/components/ClustersList.tsx:136]: yes

File: dashboard/src/API/releases.ts (hunk 2 above)
Comment (chad11111) [reply dashboard/src/API/releases.ts:292]: yes

File: dashboard/src/components/revision/RevisionResource.tsx (hunk 6 above)
Comment (chad11111) [reply dashboard/src/components/revision/RevisionResource.tsx:95]: done

File: dashboard/src/components/Badge.tsx (hunk 1 above)
Comment (chad11111) [reply dashboard/src/components/Badge.tsx:45]: cuz @undera said they are the wrong colors in slack, mind telling me which colors to use plz.

**Review by nir2002 — APPROVED** [review]

**Pull request merged**